kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -885,6 +926,114 @@ public void handleSendAndRecordButtonView(boolean isSendButtonVisible) {
         recordButton.setVisibility(showRecordButton ? (isSendButtonVisible ? View.GONE : View.VISIBLE) : View.GONE);
     }
 
+    @Override
+    public void onMessageSent(Message message) {
+    }
+
+    @Override
+    public void onMessageReceived(Message message) {
+    }
+
+    @Override
+    public void onLoadMore(boolean loadMore) {
+    }
+
+    @Override
+    public void onMessageSync(Message message, String key) {
+    }
+
+    @Override
+    public void onMessageDeleted(String messageKey, String userId) {
+    }
+
+    @Override
+    public void onMessageDelivered(Message message, String userId) {
+    }
+
+    @Override
+    public void onAllMessagesDelivered(String userId) {
+    }
+
+    @Override
+    public void onAllMessagesRead(String userId) {
+    }
+
+    @Override
+    public void onConversationDeleted(String userId, Integer channelKey, String response) {
+    }
+
+    @Override
+    public void onUpdateTypingStatus(String userId, String isTyping) {
+    }
+
+    @Override
+    public void onUpdateLastSeen(String userId) {
+    }
+
+    @Override
+    public void onMqttDisconnected() {
+    }
+
+    @Override
+    public void onMqttConnected() {
+    }
+
+    @Override
+    public void onUserOnline() {
+    }
+
+    @Override
+    public void onUserOffline() {
+    }
+
+    @Override
+    public void onChannelUpdated() {
+        if(channel==null) {
+            return;
```

Comment:
Code formatting required

Reply:
done

Reply:
it is still showing as  if(channel==null) without any spaces, thats the same at multiple places.